flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/ExecNode.java

```diff
@@ -41,9 +41,11 @@
 	String getDesc();
 
 	/**
-	 * Returns the output {@link RowType} of this node.
+	 * Returns the output {@link LogicalType} of this node,
+	 * this type should be consistent with the type parameter {@link T}.
+	 * Such as, T is RowData, the output type should be RowType.
 	 */
-	RowType getOutputType();
+	LogicalType getOutputType();
```

Comment:
You can link to JavaDoc of `RowData` for a mapping of logical types to internal data structures here.